src/workerd/server/server-test.c++

```diff
@@ -4730,6 +4730,9 @@ KJ_TEST("Server: Durable Object facets") {
                 `      results.push(await bar.increment("foo", true));  // increments bar.foo
                 `      let foo = this.ctx.facets.get("foo", {class: this.env.COUNTER});
                 `      results.push(await foo.increment(true));  // increments foo
+                `
+                `      // Delete bar, which recursively deletes its children.
+                `      this.ctx.facets.delete("bar");
```

Comment:
What happens when I have a parent facet return a child facet stub, and then we delete the parent facet, and try to use the child facet?

I assume that since we call `abort()` on both the parent and the stub will no longer be functional. Is that worth adding as a test in your opinion?